solutions/sapo-14-genghis.mdx

```diff
@@ -0,0 +1,244 @@
+---
+id: sapo-14-genghis
+source: SAPO 2014
+title: The Stables of Genghis Khan
+author: Kevin Sheng
+---
+
+## Solution
+
+Let's define $\texttt{lowestOps}[a][b]$ as the minimum number of operations to see if each horse is in the stables or not, given that we only consider horses in the interval $(\texttt{stableHorses[a]}, \texttt{stableHorses[b]})$, with $\texttt{stableHorses}$ being a sorted list of all the horses that are in the stables.
+
+Why use an open interval? It's to account for the horses with ID's that aren't contained in the range of the ID's of the horses of the stable. For example, if we had a horse with ID $4$, and the horses in the stables were $[6, 7, 10]$, then there would be no way for $\texttt{lowestOps}$ to account for that horse.
+
+To solve this, we can add two more hypothetical horses that are just out of bounds of all the other horses- one at the front and one at the end. This along with the open interval ensures that we can account for all horses excluding the hypothetical ones we've added.
```

Comment:
Replace - with a comma? It looks kind of weird

Reply:
k sure